examples/analytics_specs/main.tf

```diff
@@ -0,0 +1,20 @@
+module "atlas_basic" {
+  source = "../../"
+  org_id = var.org_id
+
+  project_name = "analytics-example-project"
+
+  cluster_name  = "AnalyticsCluster"
+  provider_name = "AWS"
+  region_name   = "US_EAST_1"
+
+  electable_specs = {
+    instance_size = "M10"
+    node_count    = 5
```

Comment:
any reason to use 5? min is 3

Reply:
No, it was just a random example. In the variable, I put as default 3.